lib/go/thrift/application_exception.go

```diff
@@ -124,6 +252,17 @@ func (p *tApplicationException) Read(ctx context.Context, iprot TProtocol) error
 					return err
 				}
 			}
+		case 3:
```

Comment:
It's likely a bad idea to write/read the additional `ValidationError` details over the wire (the implementation of `Read` and `Write` of `ValidationError`). the details should be purely local additional information for now. to be able to actually communicate that over the wire, we need to expand the spec to define the wire protocol for it.